### PR TITLE
Add elb documentation

### PR DESCRIPTION
**Pull request by freesky-edward**
This PR is to add documentation of elb service.

Resolves: #31

### CODE REVIEW

File: website/docs/r/elb_health.html.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional) The region in which to create the elb health. If
+    omitted, the `region` argument of the provider is used. Changing this
+    creates a new elb health.
+
+* `listener_id` - (Required) Specifies the ID of the listener to which the health
+    check task belongs.
```
Comment (niuzhenguo) [inline website/docs/r/elb_health.html.markdown]: better to add a "Changing this creates a new elb health" like above to let users know this argument doesn't support updating.

File: website/docs/r/elb_listener.html.markdown [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    creates a new elb listener.
+
+* `name` - (Required) Specifies the load balancer name. The name is a string
+    of 1 to 64 characters that consist of letters, digits, underscores (_), and
+    hyphens (-).
+
+* `description` - (Optional) Provides supplementary information about the listener.
+    The value is a string of 0 to 128 characters and cannot be <>.
+
+* `loadbalancer_id` - (Required) Specifies the ID of the load balancer to which
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown:57]: ditto.

File: website/docs/r/elb_listener.html.markdown [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    the listener belongs.
+
+* `protocol` - (Required) Specifies the listening protocol used for layer 4
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown]: ditto

File: website/docs/r/elb_listener.html.markdown [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    or 7. The value can be HTTP, TCP, HTTPS, or UDP.
+
+* `protocol_port` - (Required) Specifies the listening port. The value ranges from 1
+    to 65535.
+
+* `backend_protocol` - (Required) Specifies the backend protocol. If the value
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown:67]: ditto.

File: website/docs/r/elb_listener.html.markdown [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    of protocol is UDP, the value of this parameter can only be UDP. The value can
+    be HTTP, TCP, or UDP.
+
+* `backend_port` - (Required) Specifies the backend port. The value ranges from
+    1 to 65535.
+
+* `lb_algorithm` - (Required) Specifies the load balancing algorithm for the
+    listener. The value can be roundrobin, leastconn, or source.
+
+* `session_sticky` - (Optional) Specifies whether to enable sticky session.
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown:77]: ditto.

File: website/docs/r/elb_listener.html.markdown [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    The value can be true or false. The Sticky session is enabled when the value
+    is true, and is disabled when the value is false. If the value of protocol is
+    HTTP, HTTPS, or TCP, and the value of lb_algorithm is not roundrobin, the value
+    of this parameter can only be false.
+
+* `sticky_session_type` - (Optional) Specifies the cookie processing method.
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown]: ditto.

File: website/docs/r/elb_listener.html.markdown [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+    The value is insert. insert indicates that the cookie is inserted by the load
+    balancer. This parameter is valid when protocol is set to HTTP, and session_sticky
+    to true. The default value is insert. This parameter is invalid when protocol
+    is set to TCP or UDP, which means the parameter is empty.
+
+* `cookie_timeout` - (Optional) Specifies the cookie timeout period (minutes).
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown]: ditto.

File: website/docs/r/elb_listener.html.markdown [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+
+* `certificate_id` - (Optional) Specifies the ID of the SSL certificate used
+    for security authentication when HTTPS is used to make API calls. This parameter
+    is mandatory if the value of protocol is HTTPS. The value can be obtained by
+    viewing the details of the SSL certificate.
+
+* `udp_timeout` - (Optional) Specifies the UDP timeout duration (minutes). This
+    parameter is valid when protocol is set to UDP. The value ranges from 1 to 1440.
+
+* `ssl_protocols` - (Optional) Specifies the SSL protocol standard supported
```
Comment (niuzhenguo) [inline website/docs/r/elb_listener.html.markdown:117]: ditto

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+    creates a new loadbalancer.
+
+* `name` - (Required) Specifies the load balancer name. The name is a string
+    of 1 to 64 characters that consist of letters, digits, underscores (_),
+    and hyphens (-).
+
+* `description` - (Optional) Provides supplementary information about the
+    listener. The value is a string of 0 to 128 characters and cannot be <>.
+
+* `vpc_id` - (Required) Specifies the VPC ID.
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown]: ditto.

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+* `bandwidth` - (Optional) Specifies the bandwidth (Mbit/s). This parameter
+    is mandatory when type is set to External, and it is invalid when type
+    is set to Internal. The value ranges from 1 to 300.
+
+* `type` - (Required) Specifies the load balancer type. The value can be
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown:48]: ditto.

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+    Internal or External.
+
+* `admin_state_up` - (Required) Specifies the status of the load balancer.
+    Value range: 0 or false: indicates that the load balancer is stopped. Only
+    tenants are allowed to enter these two values. 1 or true: indicates that
+    the load balancer is running properly. 2 or false: indicates that the load
+    balancer is frozen. Only tenants are allowed to enter these two values.
+
+* `vip_subnet_id` - (Optional) Specifies the ID of the private network
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown:57]: ditto.

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+    to be added. This parameter is mandatory when type is set to Internal,
+    and it is invalid when type is set to External.
+
+* `az` - (Optional) Specifies the ID of the availability zone (AZ). This
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown]: ditto.

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+    parameter is mandatory when type is set to Internal, and it is invalid
+    when type is set to External.
+
+* `security_group_id` - (Optional) Specifies the security group ID. The
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown]: ditto

File: website/docs/r/elb_loadbalancer.html.markdown [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+    value is a string of 1 to 200 characters that consists of uppercase and
+    lowercase letters, digits, and hyphens (-). This parameter is mandatory
+    only when type is set to Internal.
+
+* `vip_address` - (Optional) Specifies the IP address provided by ELB.
```
Comment (niuzhenguo) [inline website/docs/r/elb_loadbalancer.html.markdown]: ditto.

**Comment by freesky-edward** [comment]
@niuzhenguo done.

**Comment by niuzhenguo** [comment]
@freesky-edward Thanks for the update!

**Comment by freesky-edward** [comment]
@paultyng could you please review this PR? thanks

**Pull request merged**